### PR TITLE
Hiroko's name was being exported as Hiro

### PR DESCRIPTION
**Pull request by lsminter**
There was a bug where Hiro had two names, Hiro and Hiroko. If you search for Hiro, three lessons would should up, if you searched for Hiroko, all of her lessons would show up. 

Screen Shot 2021-06-11 at 11 43 02 AM

Screen Shot 2021-06-11 at 12 00 13 PM

In `egghead-next/src/components/search/instructors/index.tsx`, Hiroko's name was being exported as `hiro` instead of `hiroko` and I believe that's why these were showing up. 

Screen Shot 2021-06-11 at 1 08 47 PM

After changing the name, nothing shows up after searching for `hiro`.

### CODE REVIEW

File: src/components/search/instructors/index.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const InstructorsIndex: any = {
   'john-lindquist': SearchJohnLindquist,
   'laurie-barth': SearchLaurieBarth,
   'flavio-corpa': SearchFlavioCorpa,
-  'hiro-nishimura': SearchHirokoNishimura,
+  'hiroko-nishimura': SearchHirokoNishimura,
```
Comment (lsminter) [inline src/components/search/instructors/index.tsx:19]: Just this small change.

**Comment by zacjones93** [comment]
This looks great, we need to be consistent. 

The slug in the instructor page data needs to be updated as well

https://github.com/eggheadio/egghead-next/blob/main/src/components/search/instructors/hiroko-nishimura/hiroko-nishimura-page-data.ts#L8

**Review by zacjones93 — CHANGES_REQUESTED** [review]
This looks great, we need to be consistent.

The slug in the instructor page data needs to be updated as well

https://github.com/eggheadio/egghead-next/blob/main/src/components/search/instructors/hiroko-nishimura/hiroko-nishimura-page-data.ts#L8

Also, it's company policy for all PR's to include a GIF.

File: src/components/search/instructors/hiroko-nishimura/hiroko-nishimura-page-data.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ const HirokoNishimuraPageData = [
     bio: `AWS Community Hero. Founder of AWS Newbies and LinkedIn Learning instructor.`,
     twitterHandle: 'hirokonishimura',
     websiteURL: 'https://hirokonishimura.com/',
-    slug: 'resources-by-hiro-nishimura',
+    slug: 'resources-by-hiroko-nishimura',
```
Comment (lsminter) [inline src/components/search/instructors/hiroko-nishimura/hiroko-nishimura-page-data.ts:8]: Completely missed this one. Thanks Zac.

**Review by zacjones93 — APPROVED** [review]

**Pull request merged**